app/schema/answer.py

```diff
@@ -100,3 +100,14 @@ def mandatory_error(self, state):
         state.errors = []
         state.errors.append(self.questionnaire.get_error_message('MANDATORY', self.id))
         state.is_valid = False
+
+    @staticmethod
+    def _valid_option_selected(state_input, options):
```

Comment:
Even in that case I don't believe it's logic which belongs to an answer.py. This function would only apply to radio and checkbox answers. The functionality is not dependent on state and could go in a separate file at that point otherwise putting the logic in answer.py suggests this is functionality applicable to all answers?